### DISCUSSION TITLE
Document/Demo Stereo features

### DISCUSSION THREAD

**Discussion by AndrewAnnex** [pyvista/pyvista#1583] [Ideas]
The VTK api is simply huge, and I had a quick idea to see if I can make a plotter render a scene as stereo anaglyph and quickly worked out that I could by doing the following:

```
p = pv.Plotter(notebook=True)
p.add_mesh(pv.Cube())
p.ren_win.StereoRenderOn()
p.ren_win.SetStereoTypeToAnaglyph()
p.show(jupyter_backend='static')
```
![index](https://user-images.githubusercontent.com/2126916/128905471-0b59ab69-c008-442d-911f-8a47454ae278.png)

Figured I would share how to do this here. 
I think it would be great to add some helper methods to the Plotter class to more quickly enable/disable the different stereo modes, but perhaps just documenting how to interact with the underlying vtk api more would be enough.

**Comment by akaszynski** [comment]
Nice!  Adding this as a feature request.
